src/pages/signin/SignInPageLayout/index.js

```diff
@@ -35,34 +35,48 @@ const SignInPageLayout = (props) => {
         </SignInPageContent>
     );
 
+    const hasRedirect = !_.isEmpty(backgroundStyle.redirectUri);
+
+    const graphicLayout = () => (
+        <Pressable
+            style={[
+                styles.flex1,
+                StyleUtils.getBackgroundColorStyle(backgroundStyle.backgroundColor),
+            ]}
+            onPress={() => {
+                Link.openExternalLink(backgroundStyle.redirectUri);
+            }}
+            disabled={!hasRedirect}
+        >
+            <SVGImage
+                width="100%"
+                height="100%"
+                src={backgroundStyle.backgroundImageUri}
+                resizeMode={props.isMediumScreenWidth ? 'contain' : 'cover'}
+            />
+        </Pressable>
+    );
```

Comment:
Should this be an arrow function? I think it's not necessary

Reply:
@Santhosh-Sellavel oh yes, you're right arrow function isn't necessary, also passing props.isMediumScreenWidth isn't necessary too as it is in the props so removing it where it is passed in props.